docs/examples/multimodal_classification/Multimodal_Classification_with_Ludwig_CLI.ipynb

```diff
@@ -0,0 +1,344 @@
+        "\"\"\"\n",
+        "\n",
+        "# Writes config to \"config.yaml\"\n",
+        "with open(\"config.yaml\", \"w\") as f:\n",
+        "  f.write(config_yaml)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "87VXm-aH9DYf"
+      },
+      "source": [
+        "## Create and train a model"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "background_save": true
+        },
+        "id": "fEizUlhwrSMn"
+      },
+      "outputs": [],
+      "source": [
+        "# Trains the model. This cell might take a few minutes.\n",
+        "!ludwig train --dataset twitter_human_bots_dataset.csv -c config.yaml"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "47tFtYwUuucY"
+      },
+      "source": [
+        "# Evaluate"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "WTe2wDNguw5D"
+      },
+      "outputs": [],
+      "source": [
+        "# Generates predictions and performance statistics for the test set.\n",
+        "!ludwig evaluate --model_path results/experiment_run/model \\\n",
+        "                 --dataset twitter_human_bots_dataset.csv \\\n",
+        "                 --split test \\\n",
+        "                 --output_directory results/experiment_run"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "F1VTQf6flpxU"
+      },
+      "source": [
+        "# Visualize Metrics"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "zj1ocMUtaMRJ"
+      },
+      "outputs": [],
+      "source": [
+        "# Visualizes confusion matrix, which gives an overview of classifier performance\n",
+        "# for each class.\n",
+        "!ludwig visualize --visualization confusion_matrix \\\n",
+        "                  --ground_truth_metadata results/experiment_run/model/training_set_metadata.json \\\n",
+        "                  --test_statistics results/experiment_run/test_statistics.json \\\n",
+        "                  --output_directory visualizations \\\n",
+        "                  --file_format png\n",
+        "\n",
+        "# If you run ludwig visualize locally, visualizations will automatically show in\n",
+        "# a window. Here in Colab, we can run the following code to load and display\n",
+        "# generated plots inline.\n",
+        "from IPython import display\n",
+        "import ipywidgets\n",
+        "from pathlib import Path\n",
+        "\n",
+        "ipywidgets.HBox([\n",
+        "  ipywidgets.Image(value=Path(\"visualizations/confusion_matrix__account_type_top3.png\").read_bytes()),\n",
+        "  ipywidgets.Image(value=Path(\"visualizations/confusion_matrix_entropy__account_type_top3.png\").read_bytes()),\n",
```

Comment:
This one fails  with this error message
```
FileNotFoundError: [Errno 2] No such file or directory: 'visualizations/confusion_matrix__account_type_top3.png'

```

Reply:
Woops, these filenames should be top2, not top3.  Fixed